docs/lab/02/02.md

```diff
@@ -48,17 +48,187 @@ It is the role of the *IO Bank0* to multiplex and connect the peripherals to the
 ![IO Bank0](images/gpio_mux.png)
 </div>
 
-## Bare metal
+## MMIO Development
```

Comment:
This is not MMIO, it is Hardware Access or similar.

Reply:
Done